gnovm/pkg/doc/dirs.go

```diff
@@ -70,6 +70,9 @@ func newDirs(dirs []string, modDirs []string) *bfsDirs {
 
 // tries to parse gno mod file given the filename, using Parse and Validate from
 // the gnomod package
+//
+// TODO(tb): replace by `gnomod.ParseAt` ? The key difference is the latter
```

Comment:
yes, and later, `gnomod.ParseAt` can also support new ways to guess a module when there is no `gno.mod` file.